tests/playwright/tests/db/columns/columnBarcode.spec.ts

```
@@ -8,7 +8,7 @@ interface ExpectedBarcodeData {
   barcodeSvg: string;
 }
 
-test.describe('Virtual Columns', () => {
+test.describe.only('Virtual Columns', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)